qp_shogun/shogun/shogun.py

```diff
@@ -75,7 +75,7 @@ def generate_shogun_assign_taxonomy_commands(temp_dir, parameters):
     ext = aln2ext[parameters['aligner']]
     output_fp = join(temp_dir, 'profile.tsv')
     cmds.append(
-        'shogun assign-taxonomy '
+        'shogun assign_taxonomy '
         '--aligner {aligner} '
         '--database {database} '
         '--input {input} --output {output}'.format(
```

Comment:
Would be good to add `--percent_id 0.95` to the command parameters for the sake of shotgun metagenomics (SHOGUN default 0.98 is for amplicon).